### DISCUSSION TITLE
Possible to record in H265 but stream in H264?  (...and slow camera loading?)

### DISCUSSION THREAD

**Discussion by sofakng** [blakeblackshear/frigate#6129] [General]
Is it possible to configure Frigate to record an H265 stream (direct from camera) but if I view using my web browser to transcode to H264?

I'm recording 24/7 and I'd like to record H265 to save disk space but transcode to H264 for browser playback.  Is this possible?  I think I could set the cameras to H264 and then encode H265 to disk but that would require 24/7 encoding which isn't efficient.

Also, I'm coming from Blue Iris and I noticed the web interface is much faster.  For example, when I navigate from camera to camera it's almost instant but Frigate takes 2-3 seconds before loading each camera.  (I've tried JSMPEG [which seems the fastest but no audio?], and go2rtc with webrtc/mse)

**Comment by NickM-27** [comment]
Yes, it is possible to use h265 for recording but h264 for live view. Just setup go2rtc to transcode to h264 and use that stream under `live -> stream_name` for that camera.

And, the time it takes for live view to start is based on the iframe interval of your camera since the stream has to start on an iframe. all my cameras have iframe intervals of 1 per second and live view starts very fast.

**Reply by sofakng** [reply]
Thanks so much!
